iOS/FVM/Framework/DamagedParts/Services/DamagedPartsService.swift

```diff
@@ -35,7 +35,7 @@ internal class DamagedPartsService: DamagedPartsServiceProtocol {
     
     internal func createAndGetCollectionOfDamagedParts(json: String) -> [Selection] {
         if (initialConfiguration == nil) {
-            initialConfiguration = json.replacingOccurrences(of: " ", with: "").replacingOccurrences(of: "\n", with: "")
+            initialConfiguration = json
```

Comment:
So now, add removing whitespaces when we parse JSON to damaged parts, we need it, because validation may fail because of whitespace.

Reply:
I mean "  Roof " is still a valid part. Also, cover it by tests.

Reply:
No, it may not, it is done already. What i've done is simply getting rid of clearing whitespaces from _initial_ json, which is only stored because of possibility to cancel without making changes. It does not affect processing damaged parts in any way.